### ISSUE TITLE
How can I import a pretrained FixEfficientNet model?

### ISSUE THREAD

**Issue by kopop8** [facebookresearch/FixRes#33]
Where can I find the FixEfficientNet model?
The paper says I can find a pretrained model here and I cant find it in resnext wsl either.
Am missing something?

Thanks in advance!

**Comment by xvdp** [comment]
I second this request. I cant seem to find FixEfficientNet_L2, or for that matter FixResnet50_sws
I looked at sota benchmark link in your pytorch-image-models, and I get a 403. on

urllib.request.urlretrieve('https://dl.fbaipublicfiles.com/FixRes_data/FixRes_Pretrained_Models/FixEfficientNet/FixEfficientNet_L2.pth', 'FixEfficientNet_L2.pth')
Are you guys planning to make these pretrained weights available, or am I looking in the wrong spot?

**Comment by TouvronHugo** [comment]
Hi @xvdp and @kopop8 ,
Thank you for your messages.
We will publish the links for the FixEfficientNet weights in the near future.
I'll let you know as soon as it's done.
Best,
Hugo

**Comment by xvdp** [comment]
merci! your treatment of the topic is important, not just for imagenet accuracy.

**Comment by GalSang17** [comment]
Looking forward to all the weights of efficientnet series. Thank you

**Comment by bassemkaroui** [comment]
Hi @TouvronHugo,
I see your response date from december 2020 but still I couldn't find any link to FixEfficientNet (B7) weights, so I wonder if they are still unavailable ?

**Comment by fire717** [comment]
Looking for pretrained model for FixEfficientNet-L2 at 2022....

**Comment by ErenCaldiran** [comment]
*2023

**Comment by qwerQvQ** [comment]
On the 20th day of January 2024, I still look forward to all the weights in the FixEfficientNet series. 🤗

**Comment by dnevo** [comment]
Please make it available...
